docs/Performance/promise.md

```diff
@@ -0,0 +1,195 @@
+---
+layout: default
+title: Promise
+name: Overview
+parent: Performance
+nav_order: 1
+---
+
+## Promise
+
+If you are new to Promises, I would suggest to make yourself familiar before continuing:
+
+<https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise>
+
+### Basic OData call
+
+This is a very basic OData call where the key and the callbacks are hardcoded.
+
+```js
+var model = that.getView().getModel();
+model.read("/MyEntitySet('12345')", {
+  success: function (data, response) {
+    // success handler
+  },
+  error: function (error) {
+    // error handler
+  }
+});
+```
+
+### Promisified OData call
+
+A wrapped version of the basic OData call.
+
+The key is given as parameter and the wrapping method returns a promise. The resolve / reject methods are used in the success / error callbacks to resolve or reject the Promise.
+
+As the resolve method takes only one parameter, data and response are passed as object.
+
+```js
+function readMyEntity(id) {
+  return new Promise(function (resolve, reject) {
+    var model = that.getView().getModel();
+    model.read("/MyEntitySet('" + id + "')", {
+      success: function (data, response) {
+        resolve({ data: data, response: response });
+      },
+      error: function (error) {
+        reject(error);
+      }
+    });
+  });
+}
+```
+
+The success and error handling can now be done after invoking the method.
```

Comment:
Might be helpful to the reader to briefly introduce the Promise chain "concept" here before using it.

Reply:
add a note and link to the promise chain example below